src/sr/discord_bot/bot.py

```diff
@@ -98,40 +98,64 @@ async def setup_hook(self) -> None:
 
     async def on_ready(self) -> None:
         self.logger.info(f"{self.user} has connected to Discord!")
-        guild = self.get_guild(self.guild.id)
-        if guild is None:
-            logging.error(f"Guild {self.guild.id} not found!")
+
+        if (guild := self.get_guild(self.guild.id)) is None:
+            self.logger.error("Guild not found: %r", self.guild.id)
```

Comment:
suggestion: presumably this normally indicates a configuration issue? Perhaps the error message could hint at that?

Reply:
Not necessarily. `on_ready` can be called multiple times. The first time it's a configuration issue, any subsequent could be caused by anything. I don't think we can reasonably specialize the error message here.

Reply:
Ah, ok. I wonder then if the comment text in this block might be useful to add to the error message? Just a thought.